src/sentry/receivers/onboarding.py

```diff
@@ -574,3 +576,39 @@ def record_integration_added(
                     task=task_mapping[integration_type],
                     status=OnboardingTaskStatus.COMPLETE,
                 )
+
+
+@project_transferred.connect(weak=False, dispatch_uid="onboarding.record_project_transferred")
+def record_project_transferred(old_org_id: int, updated_project: Project, **kwargs):
+
+    new_organization = Organization.objects.get(id=updated_project.organization_id)
+
+    analytics.record(
+        "project.transferred",
+        old_organization_id=old_org_id,
+        new_organization_id=new_organization.id,
+        project_id=updated_project.id,
+        platform=updated_project.platform,
+    )
+
+    existing_tasks_in_old_org = OrganizationOnboardingTask.objects.filter(
+        organization_id=old_org_id,
+        task__in=OrganizationOnboardingTask.TRANSFERABLE_TASKS,
+    )
+
+    existing_tasks_in_new_org = set(
+        OrganizationOnboardingTask.objects.filter(organization_id=new_organization.id).values_list(
+            "task", flat=True
+        )
+    )
+
+    new_tasks = [
+        task for task in existing_tasks_in_old_org if task.task not in existing_tasks_in_new_org
+    ]
+
+    for task in new_tasks:
+        task_dict = model_to_dict(task, exclude=["id", "organization", "project"])
+        copied_task = OrganizationOnboardingTask(
+            **task_dict, organization=new_organization, project=updated_project
```

Comment:
`OrganizationOnboardingTask` requires a `Project` and an `Organization` instance